pom.xml

```diff
@@ -84,29 +96,6 @@
 			<activation>
 				<jdk>[11,)</jdk>
 			</activation>
-			<build>
-				<plugins>
-					<plugin>
-						<!-- mainly so IntelliJ or other IDE's add they source directory to the IDE project -->
```

Comment:
if you read the comment, IDE's without this won't add the java11 directories as source or test source directories, so you then can work on the project

Reply:
But this breaks Maven from the commandline. Maybe there's a magic system property that only IntelliJ uses, which can be used as profile activation.

Reply:
what maven version, java version and os are you using, as this doesn't break on the command line for me

Reply:
The problem is with the duplicate class: BaseClass
`[DEBUG] -d E:\tmp-help\issue-maven-multi-release-jar-testing-master\target\classes -classpath E:\tmp-help\issue-maven-multi-release-jar-testing-master\target\classes; -sourcepath E:\tmp-help\issue-maven-multi-release-jar-testing-master\src\main\java;E:\tmp-help\issue-maven-multi-release-jar-testing-master\src\main\java11;E:\tmp-help\issue-maven-multi-release-jar-testing-master\target\generated-sources\annotations; -s E:\tmp-help\issue-maven-multi-release-jar-testing-master\target\generated-sources\annotations -g -nowarn --release 11 -encoding UTF-8 --module-version 1`

it is available in multiple source folders, which results in the following error:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /E:/tmp-help/issue-maven-multi-release-jar-testing-master/src/main/java11/com/github/nhojpatrick/issue/BaseClass.java:[3,8] duplicate class: com.github.nhojpatrick.issue.BaseClass
[INFO] 1 error
```